src/Microsoft.Azure.Batch.SoftwareEntitlement/NodeEntitlements.cs

```diff
@@ -63,7 +83,7 @@ public NodeEntitlements()
             var now = DateTimeOffset.Now;
 
             VirtualMachineId = string.Empty;
-            Created = now;
```

Comment:
Why did we delete `Created` ?

Reply:
I think it was just because it wasn't used anywhere. I can put it back if necessary. What's it for?

Reply:
> What's it for?

Compatibility, mostly. IIRC, TLSFE implementation logs that detail when an entitlement request is made. 

I'd suggest putting it back and logging it from the `Controller`.

Reply:
My mental model of `NodeEntitlements` is that it's a direct representation of the claims on a token. Is that wrong? That'd be a shame because I've found the idea hugely useful for understanding the code. But `Created` is used by neither `TokenGenerator` nor `NodeEntitlementReader`.

It did occur to me earlier (and I'd completely forgotten this) that maybe it's supposed to model the `IssuedAt` claim. If so, `TokenGenerator.Generate` could use it instead of `DateTimeOffset.Now.UtcDateTime`, and `NodeEntitlementReader` could be modified accordingly.

Reply:
Not wrong, though I'd phrase it in the reverse direction - that the token is a serialized representation of the NodeEntitlements. Essentially, though, you're right - there should be a one to one correspondence. 

Changing it to serialize to the `iat` claim would make sense.

Reply:
In that case, could we rename it to `IssuedAt`?

Reply:
Yeah, I think that would make sense.